src/Sarif/Baseline/V2/TrustMap.cs

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.CodeAnalysis.Sarif.Baseline
+{
+    /// <summary>
+    ///  TrustMap determines how much to trust different Result attributes for matching.
+    ///  
+    ///  Trust is the product of how often values match and how unique the values were.
+    ///  An attribute which always had the same value will have very low trust.
```

Comment:
>had [](start = 35, length = 3)

had => has on this line, OR matches => matched on the next line. (The present tense sounds better to me.)

Reply:
Fixed